client/landing/stepper/index.tsx

```diff
@@ -40,7 +39,6 @@ declare const window: AppWindow;
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 function initializeCalypsoUserStore( reduxStore: any, user: CurrentUser ) {
 	reduxStore.dispatch( setCurrentUser( user ) );
-	reduxStore.dispatch( requestSites() );
```

Comment:
Also see fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Syvo%2Sfhccbeg%2Shfre.cuc%3Se%3Q880s86o4%23523-og:

> We offer users a level of support based on all their plans and memberships, for example we currently give them the highest level of support offered to any of their own plan subscriptions, or sites with a paid subscription that they administrate.

Do we currently provide any customer support in the stepper?

Reply:
We have [Presales Chat](https://github.com/Automattic/wp-calypso/blob/ad33001e8bfea2df9d21b9300bd0d54e00410b66/client/lib/presales-chat/index.ts#L52) in the domain-transfer flow but I'm not sure whether it relies on all the sites or not.

Reply:
We no longer have any Happychat stuff in Calypso (thanks to @klimeryk!). I'm 90% certain Presales Chat doesn't depend on the sites because we have endpoints that fetch the sites and calculate the support tier on the backend. But @Automattic/happops-devs will be 100% sure.

Reply:
You can never be 100% sure when it comes to all the permutations in our flows ;) But you have stamp of approval from the point of view of HappOps Devs to remove this as indeed the presales chat does not rely on the current/selected site info, nor does it pass to the backend.

Reply:
Thanks for your confirmation 👍